### PR TITLE
[Improve] Improve streampark-console module base on [3.2 Constant Variables Definition]

### PR DESCRIPTION
**Pull request by wuchunfu**
## What changes were proposed in this pull request

Related issues
https://github.com/apache/incubator-streampark/issues/2986

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / no)

### CODE REVIEW

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/handler/GlobalExceptionHandler.java [hunk 1]
```diff
@@ -52,15 +52,17 @@ public class GlobalExceptionHandler {
   @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
   public RestResponse handleException(Exception e) {
     log.info("Internal server error：", e);
-    return RestResponse.fail("internal server error: " + e.getMessage(), ResponseCode.CODE_FAIL);
+    return RestResponse.fail(
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/handler/GlobalExceptionHandler.java]: We can implement a new method to make it easier to call the RestResponse.fail method. e.g:

RestResponse.fail(ResponseCode.CODE_FAIL, "internal server error: %s", e.getMessage());

Comment (wolfboys) [reply streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/handler/GlobalExceptionHandler.java]: https://github.com/apache/incubator-streampark/pull/3122

Comment (wuchunfu) [reply streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/handler/GlobalExceptionHandler.java]: Thank you for your reminder. I will modify it later

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**